catroid/src/androidTest/java/org/catrobat/catroid/test/common/LookDataTest.java

```diff
@@ -90,4 +165,63 @@ public void testCollisionInformation() {
 
 		assertEquals(expectedMetadata, metadata);
 	}
+
+	@Test
+	public void testWebPImageLookDataIsVisible() throws IOException {
+		final InputStream inputStream = baseActivityTestRule.getActivity().getResources().getAssets().open("penguin.webp");
+		File outFile = new File(android.os.Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES), "penguin.webp");
+		FileOutputStream outputStream = new FileOutputStream(outFile);
+		BitmapFactory.decodeStream(inputStream).compress(Bitmap.CompressFormat.WEBP, 100, outputStream);
+		onView(withId(R.id.button_add))
+				.perform(click());
+
+		Intent chooserResultData = new Intent();
+		Uri uri = getImageContentUri(baseActivityTestRule.getActivity(), outFile);
+		chooserResultData.setDataAndType(uri, "image/webp");
+
+		Instrumentation.ActivityResult chooserResult =
+				new Instrumentation.ActivityResult(Activity.RESULT_OK, chooserResultData);
+		intending(hasAction(Intent.ACTION_CHOOSER)).respondWith(chooserResult);
+
+		onView(withId(R.id.dialog_new_look_gallery))
+				.perform(click());
+		onView(withId(R.id.button_play))
+				.perform(click());
+		assertEquals(project.getUserVariable("color").getValue(), "#9a721c");
```

Comment:
This test fails in the pipeline

Reply:
checked and resolved the issue @sebastiangollob